terraform/outputs.tf

```diff
@@ -0,0 +1,26 @@
+output "vpc_id" {
+  value = "${module.vpc.id}"
+}
+
+output "logs_bucket_id" {
+  value = "${aws_s3_bucket.logs.id}"
```

Comment:
As far as I understand, the output *is* necessary; if I don't export this value here, don't think it's accessible when using the `terraform_remote_state` data source elsewhere. See [hashicorp/terraform#11932](https://github.com/hashicorp/terraform/issues/11932#issuecomment-298719332)